purchase_lot/views/purchase_order_view.xml

```diff
@@ -19,7 +19,7 @@
             <field name="inherit_id" ref="purchase.purchase_order_line_form2" />
                 <field name="arch" type="xml">
                     <xpath expr="//field[@name='product_id']" position="after">
-                    <field name="lot_id" />
+                    <field name="lot_id" readonly="1" />
```

Comment:
What are your thoughts on moving the readonly attribute from the Python code to the XML-records?

Reply:
Done 👍